src/pybind/mgr/cephadm/module.py

```diff
@@ -3482,7 +3483,7 @@ def tuned_profile_add_setting(self, profile_name: str, setting: str, value: str)
     def tuned_profile_rm_setting(self, profile_name: str, setting: str) -> str:
         if profile_name not in self.tuned_profiles:
             raise OrchestratorError(
-                f'Tuned profile {profile_name} does not exist. Cannot remove setting.')
+                f'Tuned profile {profile_name} does not exist. Cannot remove setting.') 
```

Comment:
this change is just adding some whitespace to the end of the line? Can we undo this one?

Reply:
Addressed in the latest commit